src/App.js

```diff
@@ -1,15 +1,23 @@
 import "./App.css";
 import Navbar from "./Components/Navbar/Navbar";
 import Footer from "./Components/Footer/Footer";
```

Comment:
suggestion: you can create index.js in the folder component and export all files there, 
then it will be in  one line and then you can import like import {Navbar, Footer} from './Components/

Reply:
That's a great suggestion thanks will Implement it for sure.!